.github/workflows/ci.yml

```diff
@@ -17,16 +17,12 @@ jobs:
       - name: Build libROM
         run: |
             mkdir ${GITHUB_WORKSPACE}/build
-            cd ${GITHUB_WORKSPACE}/build
             export CC=mpicc
             export CXX=mpicxx
-            cmake -DCMAKE_BUILD_TYPE=Debug ..
-            make
-            cmake -DCMAKE_BUILD_TYPE=Optimized ..
-            make
+            scripts/compile.sh -m
```

Comment:
Here, I'm trying to get the Github workflows to compile with the -m option, which builds MFEM as well, but it isn't working... We need this to be able to run the examples we want in the test suite.